src/CommonAssemblyInfo.cs

```diff
@@ -1,10 +1,7 @@
-using System;
 using System.Reflection;
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyCompany("Patrik Hägne")]
-[assembly: AssemblyCopyright("Copyright (c) FakeItEasy contributors. (fakeiteasyfx@gmail.com)")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en-US")]

```

Comment:
While we're at it, we should probably consider moving everything from AssemblyInfo.cs to csproj or Directory.Build.props (except `ComVisibleAttribute`, which is required by [CA1017](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1017-mark-assemblies-with-comvisibleattribute?view=vs-2017) and can't be set in csproj as far as I know)

Reply:
I think that's a good move. Probably best done now, regardless of MinVer.